prime-router/build.gradle.kts

```diff
@@ -827,7 +827,12 @@ dependencies {
         exclude(group = "org.yaml", module = "snakeyaml")
     }
     implementation("org.yaml:snakeyaml:2.2")
-    implementation("io.github.linuxforhealth:hl7v2-fhir-converter:1.0.19")
+    implementation("io.github.linuxforhealth:hl7v2-fhir-converter") {
```

Comment:
1.1.0 has been [released](https://github.com/CDCgov/prime-reportstream/pull/11925).  Dose that remove the need for version/master?

Reply:
It does not... there are some PRs that are not merged and will likely be a few more.